openapi/openapi.json

```diff
@@ -1226,46 +1625,64 @@
         "summary" : "Count",
         "description" : "Count the number of elements found in the collection(s), given the filters",
         "operationId" : "countPost",
-        "consumes" : [ "application/json;charset=utf-8" ],
-        "produces" : [ "application/json;charset=utf-8" ],
         "parameters" : [ {
           "name" : "collection",
           "in" : "path",
           "description" : "collections",
           "required" : true,
-          "type" : "string"
+          "schema" : {
+            "type" : "string"
+          }
         }, {
           "name" : "pretty",
           "in" : "query",
           "description" : "Pretty print",
-          "required" : false,
-          "type" : "boolean",
-          "default" : false
-        }, {
-          "in" : "body",
-          "name" : "body",
-          "required" : false,
           "schema" : {
-            "$ref" : "#/definitions/Count"
+            "type" : "boolean",
+            "default" : false
           }
         } ],
+        "requestBody" : {
+          "content" : {
+            "application/json;charset=utf-8" : {
+              "schema" : {
+                "$ref" : "#/components/schemas/Count"
+              }
+            }
+          }
+        },
         "responses" : {
           "200" : {
             "description" : "Successful operation",
-            "schema" : {
-              "$ref" : "#/definitions/Hits"
+            "content" : {
+              "application/json;charset=utf-8" : {
```

Comment:
will be fixed by removing the @array annotation

Reply:
ok